cmd/podman/rm.go

```diff
@@ -65,7 +62,10 @@ func rmCmd(c *cli.Context) error {
 		return err
 	}
 
-	delContainers, lastError = getAllOrLatestContainers(c, runtime, -1, "all")
+	delContainers, err := getAllOrLatestContainers(c, runtime, -1, "all")
+	if err != nil {
```

Comment:
Are we sure we want to do this? I think we want to proceed as long as some containers were retrieved